Gemfile

```diff
@@ -10,7 +10,3 @@ active_model_opts =
   end
 
 gem 'activemodel', active_model_opts
-
-platforms :rbx do
-  gem 'rubysl', '~> 2.0'
```

Comment:
Looking through history, it looks like this was added to test against Rubinius at some stage in the past?